otter/integration/tests/test_launch_stack.py

```diff
@@ -0,0 +1,143 @@
+"""
+Tests covering Otter's integration with Heat for the launch_stack launch
+configuration.
+"""
+
+from pyrsistent import pbag
+
+from twisted.internet import reactor
+from twisted.internet.defer import inlineCallbacks
+from twisted.python.log import msg
+from twisted.trial import unittest
+from twisted.web.client import HTTPConnectionPool
+
+from otter.convergence.model import get_stack_tag_for_group
+
+from otter.integration.lib.autoscale import ScalingGroup, ScalingPolicy
+from otter.integration.lib.resources import TestResources
+from otter.integration.lib.trial_tools import (
+    TestHelper,
+    get_identity,
+    get_resource_mapping,
+    region)
+
+from otter.log import log
+
+from otter.util.deferredutils import retry_and_timeout
+from otter.util.http import check_success, headers
+from otter.util.logging_treq import LoggingTreq
+from otter.util.retry import (
+    TransientRetryError,
+    repeating_interval,
+    terminal_errors_except
+)
+
+
+class TestLaunchStack(unittest.TestCase):
+    """Tests making sure launch_stack launch configurations can be used."""
+
+    def setUp(self):
+        """
+        Establish an HTTP connection pool and commonly used resources for each
+        test. The HTTP connection pool is important for maintaining a clean
+        Twisted reactor.
+        """
+        self.helper = TestHelper(self)
+        self.rcs = TestResources()
+        self.identity = get_identity(self.helper.pool)
+        self.pool = HTTPConnectionPool(reactor, False)
+        self.treq = LoggingTreq(log=log, log_response=True)
```

Comment:
Sorry for the late re-review. I just realized you don't need `self.pool` and `self.treq` as they are available in `self.helper`.